src/tools/translate-to-fuzz.h

```diff
@@ -639,8 +639,8 @@ class TranslateToFuzzReader {
           }
         }
         switch (conditions) {
-          case 0: if (!oneIn(4)) continue;
-          case 1: if (!oneIn(2)) continue;
+          case 0: if (!oneIn(4)) continue; break;
```

Comment:
I think having it all on one line like this makes this difficult to read. Could we do
```
case 0:
 if (!oneIn(4)) continue;
 break;
```
instead?

Reply:
Good point. Changed in b377dde.